docs/iot/guides/update-control-api.md

```diff
@@ -0,0 +1,134 @@
+# Tizen Update Control Native API
+
+- [Prerequisites](#prerequisites)
+- [Connecting to Server](#connecting-to-server)
+- [Checking the Latest Version](#checking-the-latest-version)
+- [Downloading and Updating to New Version](#downloading-and-updating-to-new-version)
+- [Making an Update Reservation](#making-an-update-reservation)
+- [Update Properties](#update-properties)
+
+You can control the system software version of IoT device using update control API.
+The control of system software version includes:
+- Checking the latest version
+- Downloading and updating to the latest version
+- Making a reservation for update
+
+The update procedure and the server might be different according to the platform developer.
+Therefore, this API just offers general functions for update, and platform developers must develop the application using update control API properly.
+
+>  **Note**
+>
+> As a default, the update agent works with [SmartThings Device Manager](https://console.smartthingsdm.com/) is available.
+
+## Prerequisites
+
+To enable your application to use the update control functionality:
+
+1. To use the update control API (in application), the application has to request permission by adding the following privileges to the `tizen-manifest.xml` file:
+
+```xml
+   <privileges>
+      <privilege>http://tizen.org/privilege/updatecontrol.admin</privilege>
+   </privileges>
+```
+
+2. To use the functions and data types of the update control API, include the `<update_control.h>` header file in your application:
+
+```cpp
+#include <update_control.h>
+```
+
+3. If some initialization is required to use update control functions, you can use `update_control_initialize()`:
+
+```cpp
+int error_code;
+
+error_code = update_control_initialize();
+if (error_code != UPDATE_CONTROL_ERROR_NONE)
+  return;
+```
+Likewise, there is a function `update_control_deinitialize()` for deinitialization.
+
+## Connecting to Server
+
+The new version of platform is usually uploaded at the server and each device must connect to server to get the new version.
+In some cases, connecting to server might require the procedure for getting the account information. It can be done by `update_control_initialize()` or at the each update operations (for example, **Checking the Latest Version**, **Downloading and Updating to New Version**).
```

Comment:
In some cases, connecting to server might require the account information. The account information is asked by `update_control_initialize()`, or update operations such as checking and downloading the latest version .

@sabeena21 please check the suggested sentence.

Reply:
@mijin85cho, Please find the below rephrased sentence:

In some cases, connecting to the server might require the procedure for getting the account information. The account information is obtained using `update_control_initialize()` or at the each update operations such as checking or downloading and updating  the latest version.

Reply:
@SunmLee I think sabeena21's suggestion is good. Would you please, too?

Reply:
@mijin85cho @sabeena21 Thank you for the suggestion. It looks much better. It will be replaced.